phase9/Controllers/InvertedIndexController.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Mvc;
+using SampleLibrary;
+
+namespace ASP.Controllers;
+
+[ApiController]
+[Route("[controller]/[Action]")]
+public class InvertedIndexController : ControllerBase
+{
+    private InvertedIndex _invertedIndex = new ();
+    private FileReader _fileReader = new(@"Resources");
```

Comment:
we will handle this with dependency injection in our next push

Reply:
We were going to handle it with dependency injection but unfortunately our library does not implement interfaces with both these classes